lib/judges/commands/update.rb

```diff
@@ -260,4 +264,14 @@ def include?(opts, name)
     return true if judges.empty?
     judges.any?(name)
   end
+
+  # Export churn to a file.
+  # @param [String] file The file path to write to
+  # @param [Factbase::Churn] churn The churn object
+  def churn_export(file, churn)
+    File.write(file, churn.to_s)
+    @loog.info("Churn written to #{file}: #{churn}")
+  rescue StandardError => e
+    @loog.error("Failed to write churn to #{file}: #{e.message}")
```

Comment:
@svmukhin what's the point of swallowing this exception? Let's it raise and fail the entire command.